build/cqs.tsv

```diff
@@ -189,12 +200,7 @@ org.json4s:json4s-native_2.11	3.2.10	compile
 org.locationtech.sfcurve:sfcurve-api_2.11	0.2.0	compile
 org.locationtech.sfcurve:sfcurve-zorder_2.11	0.2.0	compile
 org.locationtech.spatial4j:spatial4j	0.6	compile
-org.mortbay.jetty:jetty	6.1.26	compile
-org.mortbay.jetty:jetty-sslengine	6.1.26	compile
 org.mortbay.jetty:jetty-util	6.1.26	compile
-org.mortbay.jetty:jsp-2.1	6.1.14	compile
-org.mortbay.jetty:jsp-api-2.1	6.1.14	compile
-org.mortbay.jetty:servlet-api-2.5	6.1.14	compile
```

Comment:
turns out we didn't actually need these - bumping the maven dependency plugin version fixed a bug in how dependency trees are calculated